16_results_and_eval/iigo.tex

```diff
@@ -0,0 +1,83 @@
+
+
+\begin{figure}[h]
+    \centering
+    \subfigure[Third Simulation]{
+      \centering
+      \includegraphics[width=.48\linewidth]{16_results_and_eval/images/no_iigo_1.png}
+    }
+    \subfigure[Sixth Simulation]{
+      \centering
+      \includegraphics[width=.48\linewidth]{16_results_and_eval/images/no_iigo_2.png}
+    }
+    \caption{Unpredictability of outcomes in No-IIGO simulations}
+    \label{fig:ResultsAndEval:no_iigo_unpredictable}
+\end{figure}
+
+Both simulations of the same experimental setup, Figures ~\ref{fig:ResultsAndEval:no_iigo_unpredictable}(a) and ~\ref{fig:ResultsAndEval:no_iigo_unpredictable}(b)) show a depleted Common Pool (light green line) with close to 0 resources throughout the entire simulation. While all agents were able to survive the full 100 turns in the former, only one was able to make it past the 30th turn in the latter, eventually dying at the 60th turn. 
+
+The same metrics used to quantify the Baseline results in Table ~\ref{table:Simulations:num_metric} were monitored and aggregated for this experiment, as well as ten simulation of the Baseline configuration (with IIGO allowed to proceed at the usual cost). Table ~\ref{table:ResultsAndEval:iigo_vs_no_iigo} shows the results obtained:
+
+\begin{table}[h]
+    \centering
+    \begin{tabular}{|l|c|c|c|c|}
+    \hline
+    \textbf{Metric}                     & \multicolumn{2}{c}{\textbf{Mean}}    & \multicolumn{2}{c}{\textbf{Standard Deviation}}   \\ \hline
+                                        & \textbf{IIGO}      & \textbf{No IIGO}      &\textbf{IIGO}       & \textbf{No IIGO}                 \\ 
+    \textbf{Archipelago Survivability}  & 100 & 93 & 0 & 16.36     \\
+    \textbf{First Island Death}         & 100 & 55.2 & 0 & 42.64    \\
+    \textbf{Gini Index}                 & 0.39 & 0.59 & 0.024 & 0.11   \\
+    \textbf{Disasters Survived}         & 20 & 18.4 & 0 & 3.66    \\
+    \textbf{ADDM}                       & 332.7 & 86.4 & 81.77 & 48.27    \\
+    \textbf{AFS}                        & 1.91 & 1.57 & 0.11 & 0.17    \\ \hline
+\end{tabular}
+\caption{Metrics across ten simulations, with and without IIGO}
+\label{table:ResultsAndEval:iigo_vs_no_iigo}
+\end{table}
+
+
+\subsection{Analysis of No-IIGO Simulation Results}
+\label{subsec:ResultsAndEval:no-iigo:analysis_non_iigo_results}
+
+\subsubsection{Comparison of Means}
+\label{subsec:ResultsAndEval:no-iigo:comparison_of_means}
+Comparing the means of the computed metrics shows a clear deterioration in performance of the agents when the IIGO does not run for the entire simulation. 
+On average, an agent dies halfway through the simulation (turn 55) and the archipelago is unable to survive past 93 turns. 
+
+Wealth is also far less equally distributed; the Average Gini Index is 50\% higher for the simulations without the IIGO. This can be attributed to the shorter lifespan of poorer islands, and the absence of the proportional tax system that is typically implemented in IIGO sessions. 
```

Comment:
```suggestion
Wealth is also observed to be far less equally distributed. The Average Gini Index is $50\%$ higher for the simulations without the IIGO. This can be attributed to the shorter lifespan of poorer islands, and the absence of the proportional tax system that is typically implemented in IIGO sessions. 
```